src/dmangle.d

```diff
@@ -630,17 +629,17 @@ public:
          * -0X1.1BC18BA997B95P+79   => N11BC18BA997B95P79
          * 0X1.9P+2                 => 19P2
          */
-        if (Port.isNan(value))
+        if (isNaN(value))
```

Comment:
> Why cares, they all do the same thing?

One would hope so, but I've seen enough bad implementations of those isXXXX floating point functions that I don't trust pulling some random one from somewhere. That goes for all the math functions, too. Turns out that C library programmers tend to stink at floating point.

I once attended a talk by a physics prof on doing numerical work. He was astonished when I told him the C library sin(), cos(), etc., could not be trusted. He didn't believe me. I know from bitter experience. The usual faults they have are truncated precision and erratic handling of special values.

So, in code I need to trust, I need to see **which** implementation is being called.